frontend/src/lang/pl-PL.json

```
@@ -0,0 +1,94 @@
+    "Create": "Utwórz",
+    "signedInDisp": "Zalogowany jako {0}",
+    "signedInDispDisabled": "Autoryzacja wyłączona.",
+    "home": "Strona główna",
+    "console": "Konsola",
+    "registry": "Rejestr",
+    "compose": "Stwórz",
+    "addFirstStackMsg": "Stwórz swój pierwszy stos!",
+    "stackName" : "Nazwa stosu",
+    "deployStack": "Wdroż",
+    "deleteStack": "Usuń",
+    "stopStack": "Zatrzymaj",
+    "restartStack": "Uruchom ponownie",
+    "updateStack": "Aktualizuj",
+    "startStack": "Uruchom",
+    "editStack": "Edytuj",
+    "discardStack": "Odrzuć",
+    "saveStackDraft": "Zapisz",
+    "notAvailableShort" : "N/D",
+    "deleteStackMsg": "Czy na pewno chcesz usunąć ten stos?",
+    "stackNotManagedByDockgeMsg": "Ten stos nie jest zarządzany przez Dockge.",
+    "primaryHostname": "Podstawowa nazwa hosta",
+    "general": "Ogólne",
+    "container": "Kontener | Kontenery",
+    "scanFolder": "Skanuj folder ze stosami",
+    "dockerImage": "Obraz",
+    "restartPolicyUnlessStopped": "Chyba że zatrzymano",
+    "restartPolicyAlways": "Zawsze",
+    "restartPolicyOnFailure": "Przy awarii",
+    "restartPolicyNo": "Nie",
+    "environmentVariable": "Zmienna środowiskowa | Zmienne środowiskowe",
+    "restartPolicy": "Polityka restartu",
+    "containerName": "Nazwa kontenera",
+    "port": "Port | Porty",
+    "volume": "Wolumen | Wolumeny",
+    "network": "Sieć | Sieci",
+    "dependsOn": "Zależność kontenera | Zależności kontenera",
+    "addListItem": "Dodaj {0}",
+    "deleteContainer": "Usuń",
+    "addContainer": "Dodaj kontener",
+    "addNetwork": "Dodaj sieć",
+    "disableauth.message1": "Czy na pewno chcesz <strong>wyłączyć uwierzytelnianie</strong>?",
+    "disableauth.message2": "Przeznaczone dla scenariuszy <strong>gdzie zamierzasz zaimplementować uwierzytelnianie zewnętrzne</strong> przed Dockge, takie jak Cloudflare Access, Authelia lub inne mechanizmy uwierzytelniania.",
+    "passwordNotMatchMsg": "Powtórzone hasło nie pasuje.",
+    "autoGet": "Automatyczne pobieranie",
+    "add": "Dodaj",
+    "Edit": "Edytuj",
+    "applyToYAML": "Zastosuj do YAML",
+    "createExternalNetwork": "Utwórz",
+    "addInternalNetwork": "Dodaj",
+    "Save": "Zapisz",
+    "Language": "Język",
+    "Current User": "Aktualny użytkownik",
+    "Change Password": "Zmień hasło",
+    "Current Password": "Aktualne hasło",
+    "New Password": "Nowe hasło",
+    "Repeat New Password": "Powtórz nowe hasło",
+    "Update Password": "Aktualizuj hasło",
+    "Advanced": "Zaawansowane",
+    "Please use this option carefully!": "Proszę używać tej opcji ostrożnie!",
+    "Enable Auth": "Włącz autoryzację",
+    "Disable Auth": "Wyłącz autoryzację",
+    "I understand, please disable": "Rozumiem, proszę wyłączyć",
+    "Leave": "Opuść",
+    "Frontend Version": "Wersja frontu",
+    "Check Update On GitHub": "Sprawdź aktualizacje na GitHub",
+    "Show update if available": "Pokaż aktualizację, jeśli dostępna",
```

```suggestion
    "Show update if available": "Pokaż aktualizacje, jeśli dostępne",
```
plural for consistency